pyramid_oereb/standard/pyramid_oereb.yml.mako

```diff
@@ -316,9 +316,9 @@ pyramid_oereb:
           provider:  pyramid_oereb.standard.hook_methods.get_surveying_data_provider
     general_information:
       text:
```

Comment:
has to be removed to get it working - It's done by PR https://github.com/camptocamp/pyramid_oereb/pull/396